### PR TITLE
[QA-15634] - Deleting mail on android prevents student from exiting mailer

### PR DESCRIPTION
**Pull request by ProdigyChrisLight**
## Description
Fix for QA-15634 - Deleting mail on android prevents student from exiting mailer

## Checklist:
- [x] I reviewed my code before submitting
- [x] I verified my code follows our style guidelines
- [x] I created and/or updated documentation where applicable (JSDocs, Confluence, etc.)
- [x] I created and/or updated unit tests where applicable
- [x] I determined if this feature requires downtime

### CODE REVIEW

File: src/input/Input.js [hunk 1]
```diff
@@ -969,6 +969,41 @@ Phaser.Input.prototype = {
...
+        {
+            return this.pointer1.releaseActive(event);
+        }
+
+        if (this.pointer2.active && this.pointer2.identifier === event.identifier)
+        {
+            return this.pointer2.releaseActive(event);
+        }
+
+        for (var i = 2; i < this.pointers.length; i++)
```
Comment (jamieallen18) [inline src/input/Input.js]: Could you document why this starts at 2?

Comment (ProdigyChrisLight) [reply src/input/Input.js]: The code was copied from an existing Phaser function. This has been refactored as an optional parameter to the existing functions (stop, stopPointer).

**Comment by samme** [comment]
Could you resolve conflicts?

**Comment by ProdigyChrisLight** [comment]
Apologies. This request was meant for a different fork of phaser-ce.

**Pull request closed without merging**